src/renderer/components/dex/OrderForm.vue

```diff
@@ -78,6 +77,7 @@
       :onConfirmed="orderConfirmed" :onCancel="hideOrderConfirmationModal"></aph-order-confirmation-modal>
     <aph-deposit-withdraw-modal v-if="$store.state.depositWithdrawModalModel"
       :onConfirmed="depositWithdrawConfirmed" :onCancel="hideDepositWithdrawModal"></aph-deposit-withdraw-modal>
+    <aph-spinner v-if="!$store.state.holdings.length" identifier="fetchHoldings"></aph-spinner>
```

Comment:
So does this make the spinner show in the center of the entire order form area while holdings are loading?

Reply:
Yes - let me add a screen shot for that scenario.

Reply:
Added.